src/Unosquare.Labs.EmbedIO/IWebModule.cs

```diff
@@ -30,6 +30,14 @@ public interface IWebModule
         /// The handlers.
         /// </value>
         ModuleMap Handlers { get; }
+        
+        /// <summary>
+        /// Gets the server owning this module. This property is set automatically after registering the module.
+        /// </summary>
+        /// <value>
+        /// The server.
+        /// </value>
+        WebServer Server { get; set; }
```

Comment:
should be only gettable?

Reply:
Nop, by design with inject the server in the module.